healvis/observatory.py

```diff
@@ -113,40 +113,66 @@ def set_pointings(self, time_arr):
         """
         Set the pointing centers (in ra/dec) based on array location and times.
             Dec = self.lat
+        Also sets the north pole positions in ICRS.
         RA  = What RA is at zenith at a given JD?
         """
         telescope_location = EarthLocation.from_geodetic(self.lon * units.degree, self.lat * units.degree)
+        self.telescope_location = telescope_location
```

Comment:
This happens a bit throughout `healvis`, and I don't want to suggest to change everything, but I thought I would mention that I find it really hard to follow code that sets instance attributes outside of `__init__`

Reply:
Good point. I've moved that particular assignment to the init function, and added some more documentation on other attributes, even if they're not set there.